schema/codelists/interestType.csv

```diff
@@ -1,22 +1,24 @@
-rightsGrantedByContract,Rights granted by contract,"An interest that is granted by contract.",
-conditionalRightsGrantedByContract,Conditional rights granted by contract,"An interest that exists only if some contractual condition is met.",
-controlViaCompanyRulesOrArticles,Control via company rules or articles,"Control of an entity gained through a provision in company articles or by shareholder agreement.",
+beneficiaryOfLegalArrangement,Beneficiary of a legal arrangement,A person who benefits from a trust or other legal arrangement's holdings or activities.,
+rightsToSurplusAssetsOnDissolution,Rights to surplus assets on dissolution,The right to a share in the amount of an asset or resource that exceeds the portion that is utilized upon the winding up of an entity.,
+rightsToProfitOrIncome,Rights to receive profits or income,An economic interest in an entity. Defined as beneficial ownership rights beyond those otherwise implied by ownership structures that are granted by contract.,
+rightsGrantedByContract,Rights granted by contract,An interest that is granted by contract.,
+conditionalRightsGrantedByContract,Conditional rights granted by contract,An interest that exists only if some contractual condition is met.,
+controlViaCompanyRulesOrArticles,Control via company rules or articles,Control of an entity gained through a provision in company articles or by shareholder agreement.,
 controlByLegalFramework,Control by legal framework,"Control of an entity gained through a legal framework, such as a combination of legislation (primary and secondary). This type of interest is created by governments and legislators in order to establish and guide agencies and entities linked to the state.",
-boardMember,Board member,"One of a group of people constituted as the strategic decision-making body of an organization.",
-boardChair,Board chair,"The person holding the most power and authority on the board of directors.",
+boardMember,Board member,One of a group of people constituted as the strategic decision-making body of an organization.,
+boardChair,Board chair,The person holding the most power and authority on the board of directors.,
 unknownInterest,Unknown interest,"The interestedParty is known to have an interest in the subject of this Relationship Statement, but the nature of the interest is unknown.",
-unpublishedInterest,Unpublished interest,"The nature of this interest is known but is not published.",
-enjoymentAndUseOfAssets,Enjoyment and use of assets,"The use of assets belonging to an entity.",
+unpublishedInterest,Unpublished interest,The nature of this interest is known but is not published.,
+enjoymentAndUseOfAssets,Enjoyment and use of assets,The use of assets belonging to an entity.,
 rightToProfitOrIncomeFromAssets,Right to profit or income from assets,"The right to derive profits, income or both from assets belonging to an entity.",
+nominee,Nominee,A person who acts on behalf of the nominator in a nominee arrangement.,
```

Comment:
```suggestion
nominee,Nominee,A person who agrees to act on behalf of the nominator in a specified capacity.,
```
I think 'nominee arrangement' is a bit BODS-centred as terminology goes. So maybe avoid? (In English, I suspect 'nominee agreement' is the most common terminology. Either way, I think this phrasing works.)